3-semantic-html/basic.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <meta charset="utf-8">
+    <title>Title</title>
+  </head>
+  <body>
+      <main>
+        <article>
+            <header>
+                <h1>Title</h1>
+                <nav>
+                    <li>Element 1</li>   
+                    <li>Element 2</li>   
+                    <li>Element 3</li>   
+                </nav>
+            </header>
+            <section>
+            </section>
+                <p>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                </p>
+            <footer>
+                <p>
+                    Company Information 2020
+                </p>
+                <p>
+                    Helmut David Soler
+                </p>
+                <nav>
+                    <li>Link 1</li>
+                    <li>Link 2</li>
+                    <li>Link 3</li>
+                </nav>
+            </footer>
+        </article>
+      </main>
```

Comment:
Look at the layout of the exercise here: https://github.com/pushdev-code/html-tutorial/blob/master/3-semantic-html/index.md.  I see that you are nesting all the tags inside the `<main>`
Please use the tags that we are asking for the basic structure. Inside that elements you can put the content that you want. Something like this:
```
<body>
    <header>
        <nav>
            <!-- your content -->
        </nav>
    </header>
    <main>
        <!-- here you have to include the <article> tag as it's in the tree (DOM) image -->
    </main>
    <aside>
        <!-- your content -->
    </aside>
    <footer>
        <nav>
            <!-- your content -->
        </nav>
    </footer>
</body>
```